app/routes/bdb/components/BdbDetail.js

```diff
@@ -237,6 +239,21 @@ export default class BdbDetail extends Component<Props, State> {
             </td>
             <td>{truncateString(event.location, 50)}</td>
             <td>{truncateString(event.description, 70)}</td>
+            <td>
+              {event.survey && (
+                <Tooltip content="Spørreundersøkelse">
+                  <Link to={`surveys/${event.survey}`}>
```

Comment:
This becomes a relative link, instead of an absolute one.

By following link:
<img width="243" alt="Screenshot 2019-11-11 at 12 29 00" src="https://user-images.githubusercontent.com/14221386/68584102-fb7ec100-047e-11ea-8372-0ebe0a954d98.png">
<img width="1126" alt="Screenshot 2019-11-11 at 12 29 03" src="https://user-images.githubusercontent.com/14221386/68584113-00437500-047f-11ea-945b-b4e70ad9dc46.png">


Actual survey, found by navigating there manually:
<img width="206" alt="Screenshot 2019-11-11 at 12 29 09" src="https://user-images.githubusercontent.com/14221386/68584114-00437500-047f-11ea-9a8a-efcd640b095b.png">
<img width="1180" alt="Screenshot 2019-11-11 at 12 29 13" src="https://user-images.githubusercontent.com/14221386/68584115-00dc0b80-047f-11ea-8f63-031cecf26125.png">

I believe this is fixed by just changing the link to `/surveys/...`, ie adding an initial slash.

Reply:
Good catch!